openedx/core/djangoapps/team_api/views.py

```diff
@@ -0,0 +1,407 @@
+            only through merge patch.
+
+        **Example Requests**:
+
+            GET /api/team/v0/teams/{team_id}}
+
+            PATCH /api/team/v0/teams/{team_id} "application/merge-patch+json"
+
+        **Response Values for GET**
+
+            If the user is logged in, the response contains the following fields:
+
+            * id: The team's unique identifier.
+
+            * name: The name of the team.
+
+            * is_active: True if the team is currently active. If false, the team is considered "soft deleted" and
+              will not be included by default in results.
+
+            * course_id: The identifier for the course this team belongs to.
+
+            * topic_id: Optionally specifies which topic the team is associated with.
+
+            * date_created: Date and time when the team was created.
+
+            * description: A description of the team.
+
+            * country: Optionally specifies which country the team is associated with.
+
+            * language: Optionally specifies which language the team is associated with.
+
+            * membership: A list of the users that are members of the team. See membership endpoint for more detail.
+
+            For all text fields, clients rendering the values should take care
+            to HTML escape them to avoid script injections, as the data is
+            stored exactly as specified. The intention is that plain text is
+            supported, not HTML.
+
+            If the user is not logged in, a 403 error is returned.
+
+            If the specified team does not exist, a 404 error is returned.
+
+        **Response Values for PATCH**
+
+            Only staff can patch teams. If the specified team does not exist, a
+            404 error is returned.
+
+            If "application/merge-patch+json" is not the specified content type,
+            a 415 error is returned.
+
+            If the update could not be completed due to validation errors, this
+            method returns a 400 error with all error messages in the
+            "field_errors" field of the returned JSON.
+    """
+
+    authentication_classes = (SessionAuthenticationAllowInactiveUser, OAuth2AuthenticationAllowInactiveUser)
+    permission_classes = (permissions.IsAuthenticated, IsStaffOrReadOnly)
+    lookup_field = 'team_id'
+    serializer_class = CourseTeamSerializer
+    parser_classes = (MergePatchParser,)
+
+    def get_queryset(self):
+        """
+        Returns the queryset used to access the given team.
+        """
+        return CourseTeam.objects.all()
+
+
+class TopicListView(APIView):
+    """HTTP endpoint for retreiving a list of topics for a given course."""
+
+    authentication_classes = (SessionAuthenticationAllowInactiveUser,)
+    permission_classes = (permissions.IsAuthenticated,)
+
+    page_size = 10
+    # TODO determine the correct max page size
+    max_page_size = sys.maxint
```

Comment:
I don't see max_page_size being specified in other APIs. Check with the API hipchat room.